fix_ui.py

```
@@ -0,0 +1,26 @@
+import re
+
+with open("tests/test_flow_studio_ui_ids.py", "r") as f:
```

Ruff (0.15.9)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)